packages/marko-web-inquiry/browser/default-form.vue

```diff
@@ -180,7 +205,24 @@ export default {
     country: '',
     postalCode: '',
     comments: '',
+    gdprOptIn: false,
   }),
+  computed: {
+    gdpr() {
+      const gdprCountries = ['AT', 'BE', 'BG', 'HR', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'HU', 'IS', 'IE', 'IT', 'LV', 'LI', 'LT', 'LU', 'MT', 'NL', 'NO', 'PL', 'PT', 'RO', 'SK', 'ES', 'SE', 'GB'];
```

Comment:
This value should respect the `isGdprEnabled` boolean and return false if disabled. This list of companies should be overridable -- make it the default value for a `gdprCountryCodes` property.